web/client/components/app/StandardRouter.jsx

```diff
@@ -17,10 +17,13 @@ import Localized from '../I18N/Localized';
 import Theme from '../theme/Theme';
 import {ErrorBoundary} from 'react-error-boundary';
 import ErrorBoundaryFallbackComponent from './ErrorFallBackComp';
+import assign from 'object-assign';
 
 const ThemeProvider = connect((state) => ({
     theme: state.theme?.selectedTheme?.id
-}))(Theme);
+}), {}, (stateProps, dispatchProps, ownProps) => {
```

Comment:
Does this make theme in ownProps win over the state? Did I understood the fix correctly?

Reply:
yes

Reply:
good for me, then